data/experiment-recipe-samples/rollout-mobile.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Also figure out a minimum supported version from mobile which is tricky becuase it varies by application (ie fenix/ios/focus/klar/etc)